contracts/libraries/DnGmxJuniorVaultManager.sol

```diff
@@ -322,13 +323,17 @@ library DnGmxJuniorVaultManager {
         int256 borrowVal = borrowValue.toInt256();
 
```

Comment:
ref: https://github.com/RageTrade/delta-neutral-gmx-vaults/blob/a2107d37b789494454bd4ede7d217d8723474de4/contracts/libraries/DnGmxJuniorVaultManager.sol#L626-L629

can there be case when diff < threshold so conversion didn't took place and hence repayment to senior vault fails?

Reply:
This case would only occur close to the last deposits being withdrawn, since otherwise there would be enough extra assets to give back to the withdrawer. In this case we can reduce the threshold to a lower value to resume normal functioning.